OpenSRP/opensrp-client-tasking
# Fix reference and integration issues

Depends on https://github.com/OpenSRP/opensrp-client-tasking/pull/19


## To test

1. Run `./gradlew clean && ./gradlew :opensrp-tasking:uploadArchives -PmavenLocal=true --stacktrace --rerun-tasks` in the root folder of the project to publish the lib in your local maven repository
2. Run `ls ~/.m2/repository/org/smartregister/opensrp-client-tasking/` to confirm that the folder `0.0.3-SNAPSHOT` was created & artifact published
3. Update your Reveal project & switch to the `integrate-tasking-library` branch in this [PR](https://github.com/OpenSRP/opensrp-client-reveal/pull/931)
4. Update your oauth configs in `local.properties` & run the project
5. Login, creating new structures among other features should work
